rules/windows/defense_evasion_stop_process_service_threshold.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/12/03"
+maturity = "production"
+updated_date = "2020/12/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule identifies a high number (10) of process modifications (stop, delete, or suspend) from the same host within a
+short time period. This may indicate a defense evasion attempt.
+"""
+from = "now-9m"
```

Comment:
So for this guy it's 10 terminations in 9 minutes?  I wonder if we will run into false positives if it's open for too long. Most of the ransomware terminations are happening within milliseconds/seconds of each other. But maybe it's not performant on ES to set it up like that.

Reply:
in this case its mainly about avoiding performance impact of doing a query every 1m (we can convert it to sequence)